### PR TITLE
Added sen2 config

### PR DESCRIPTION
**Pull request by RaymondMoo**

### CODE REVIEW

**Comment by cjrd** [comment]
Did you verify the data is in the expected format/range/etc?

**Comment by cjrd** [comment]
Please create a similar proof-of-correctness =)
[grace-fo-plot.ipynb](https://github.com/bair-climate-initiative/metaearth/blob/main/nbs/grace-fo-plot.ipynb)

**Comment by RaymondMoo** [comment]
Added a test notebook sen2-plot.ipynb

File: setup.cfg [hunk 1]
```diff
@@ -73,6 +73,8 @@ tests =
     pytest>=6.1.2,<8
     # pytest-cov 2.4+ required for pytest --cov flags
     pytest-cov>=2.4,<4
+    # rasterio required for reading GeoTIFF
+    rasterio>=1.3,<4
```
Comment (cjrd) [inline setup.cfg]: move under tests

File: nbs/sen2-plot.ipynb [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "37cc87aa",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
```
Comment (RitwikGupta) [inline nbs/sen2-plot.ipynb]: It looks like the notebook fails to plot the data

**Review by RitwikGupta — CHANGES_REQUESTED** [review]

File: nbs/sen2-plot.ipynb (hunk 2 above)
Comment (RaymondMoo) [reply nbs/sen2-plot.ipynb]: Fixed.

**Comment by RaymondMoo** [comment]
Moved to different pull request.

**Pull request closed without merging**